### ISSUE TITLE
No Requests logged since updating

### ISSUE THREAD

**Issue by andykent** [ProxymanApp/Proxyman#1834]
## Description
I'm a long time ProxyMan customer and things have been fairly stable for me but a recent update. (4.14 or possibly 4.13) broke request logging completely for me and am no longer seeing any data.
 
## Steps to Reproduce

1. Launch Proxyman
2. Make HTTP requests from Mac
3. Nothing is recoded in Proxyman UI

## Current Behavior

I don't see any requests logged but the titlebar does say "Listening on..." and the status bar does show up/down network traffic so something is listening somewhere I think.

I _think_ the issue may have started when I was prompted to install the new 1.5.0 HelperTool so as a first step I tried deleting and reinstalling the HelperTool, it re-installed successfully (as 1.5.0) but made no difference to the issue.

## Expected Behavior

HTTP requests show up in the UI as previously.

## Environment

- App version: 4.14
- macOS version: 14.1

**Comment by NghiaTranUIT** [comment]
Sorry for this trouble.

Can you try do delete this file at /Library/PrivilegedHelperTools/com.proxyman.NSProxy.HelperTool and install the Helper Tool again?

**Comment by NghiaTranUIT** [comment]
It seems Proxyman doesn't connect to the Helper Tool, so Proxyman can't override the system proxy.

**Comment by andykent** [comment]
Thanks for the fast response.
As mentioned I had already tried that but I just tried it again with the same results. 

One thing I did notice is that it has a warning about `networksetup cli` I'm not sure what would be causing that. Maybe it's related?

I ran `sudo rm /Library/PrivilegedHelperTools/com.proxyman.NSProxy.HelperTool` and then did the steps in the screenshots below...

<img width="601" alt="Screenshot 2023-10-31 at 12 20 14" src="https://github.com/ProxymanApp/Proxyman/assets/614/acb23dec-05e0-4407-81ce-9c916095bcbd">

<img width="641" alt="Screenshot 2023-10-31 at 12 20 23" src="https://github.com/ProxymanApp/Proxyman/assets/614/d481f8f1-9429-4214-ae3f-5036f913631c">

<img width="599" alt="Screenshot 2023-10-31 at 12 20 34" src="https://github.com/ProxymanApp/Proxyman/assets/614/938ae821-eb91-442f-a854-b048e0d11702">

**Comment by NghiaTranUIT** [comment]
Then, it's working right? Your last screenshot shows the Helper Tool v1.5 is installed.

You can try toggle the proxy in Tools Memu -> Proxy Settings-> Override system proxy

**Comment by andykent** [comment]
The installation worked but the requests are still not shown. This is what I see in the main window.

<img width="1305" alt="Screenshot 2023-10-31 at 12 29 06" src="https://github.com/ProxymanApp/Proxyman/assets/614/aa84890f-6b34-4681-a510-e4a8386980d2">

I just tried toggling the "Override macOS Proxy" off and on again but it made no difference.

**Comment by NghiaTranUIT** [comment]
You might check if there is any VPN app is ON. Or conflict with Security apps in your Macbook.

Try to turn it off will fix it

**Comment by andykent** [comment]
NordVPN

**Comment by Geczy** [comment]
my issue was leaving the VPN on too 😂, this was bugging me, finally it works. thanks!

**Comment by NghiaTranUIT** [comment]
Thanks. I guess I can implement a small feature to prevent it in the future:
1. After launch 5 seconds,
2. If there is no requests in the main table even though the Overridden Proxy is enabled
3. Get all [running apps](https://developer.apple.com/documentation/appkit/nsworkspace/1534059-runningapplications) and check if there is any VPN app is active 
4. Show the small ⚠️ in the status bar -> Notify the user might turn off these VPN apps.

How do you think 🤔

**Comment by Geczy** [comment]
Yeah that seems okay. But if there a more intelligent way to know if a vpn is enabled than to check list of apps? There are thousands of vpns. I use one called globalprotect which is for work. Like something with dns check? Or location check

**Comment by NghiaTranUIT** [comment]
It's a good point. I suppose that I can use the Network framework to check if there is any VPN is active, and then try to get the name.
